docs/recipes/create-cluster-agent.mdx

```diff
@@ -0,0 +1,105 @@
+---
+sidebar_position: 7
+---
+
+import EmbedCard from "@site/src/components/EmbedCard";
+
+# Create Cluster Agent
+
+Cluster Proxy and Cluster Agents offer a solution to verify ownership when referencing a [cluster](/glossary/terms#cluster) during simultaneous spore creation.
+
+You will learn how to create a [Cluster Agent](/glossary/terms#cluster-agent), which serves as a representative for a Cluster. This enables the creation of [clustered spores](/recipes/create-clustered-spore) without requiring the signatures or approvals from the owner of the Cluster being represented.
+
+## Create a Cluster Agent
+
+### Method 1
+
+You can create a Cluster Agent by referencing a [ClusterProxy](/recipes/create-cluster-proxy) with the [`createClusterAgent` API](https://github.com/sporeprotocol/spore-sdk/blob/beta/docs/core/composed-apis.md#createclusteragent) from spore-sdk:
```

Comment:
@ShookLyngs this doc from `spore-sdk` repo is outdated or not? please confirm.

Reply:
The composed-apis doc should be up-to-date, and it should be updated if it's not.

Reply:
> The composed-apis doc should be up-to-date, and it should be updated if it's not.

the 'should' means 'already'?